frontend/src/views/omni/Modals/DownloadInstallationMedia.vue

```diff
@@ -50,6 +50,60 @@ import { showError, showSuccess } from '@/notification'
 import ExtensionsPicker from '@/views/omni/Extensions/ExtensionsPicker.vue'
 import CloseButton from '@/views/omni/Modals/CloseButton.vue'
 
+const supportsOverlay = (talosVersion?: string) =>
+  !!talosVersion && semver.gte(talosVersion, '1.7.0')
+
+const getImageFactoryPXEBaseURL = (baseURL: string): string => {
+  const url = new URL(baseURL)
+
+  url.host = `pxe.${url.host}`
+
+  return url.toString()
+}
```

Comment:
is this always true though? even if factory is configured to be something other than `factory.talos.dev`? should we return it as part of the features spec from backend like we do the factory URL?